client/app/components/packages/rwcds-form/show.hbs

```diff
@@ -0,0 +1,249 @@
+        Analysis
+      </h2>
+
+      <p class="">
+        <strong>
+          Description of development assumptions:
+        </strong>
+        <br />
+        {{@package.rwcdsForm.dcpDevelopmentsiteassumptions}}
+      </p>
+
+      <h3>
+        No-Action Scenario
+      </h3>
+
+      <p class="grid-x">
+        <strong class="cell auto large-padding-right">
+          Is the No-Action scenario the same as the Existing Conditions?
+        </strong>
+        <span class="cell large-3">
+          {{if @package.rwcdsForm.dcpExistingconditions "Yes" "No" }}
+        </span>
+      </p>
+
+      {{#if @package.rwcdsForm.dcpDescribethenoactionscenario}}
+        <p class="ruled-top">
+          <strong>
+            Description of No-Action scenario:
+          </strong>
+          <br />
+          {{@package.rwcdsForm.dcpDescribethenoactionscenario}}
+        </p>
+      {{/if}}
+
+      {{#if @package.rwcdsForm.dcpHowdidyoudeterminethenoactionscenario}}
+        <p class="ruled-top">
+          <strong>
+            How No-Action scenario was determined:
+          </strong>
+          <br />
+          {{@package.rwcdsForm.dcpHowdidyoudeterminethenoactionscenario}}
+        </p>
+      {{/if}}
+
+      <h3>
+        With-Action Scenario
+      </h3>
+
+      <p class="grid-x">
+        <strong class="cell auto large-padding-right">
+          Is the proposed project the same as the proposed With-Action scenario?
+        </strong>
+        <span class="cell large-3">
+          {{if @package.rwcdsForm.dcpIsrwcdsscenario "Yes" "No" }}
+        </span>
+      </p>
+
+      <p class="ruled-top">
+        <strong>
+          Why the project is the same as the propsed With-Action scenario:
+        </strong>
+        <br />
+        {{@package.rwcdsForm.dcpRwcdsexplanation}}
+      </p>
+
+      <p class="ruled-top">
+        <strong>
+          Description of With-Action scenario:
+        </strong>
+        <br />
+        {{@package.rwcdsForm.dcpDescribethewithactionscenario}}
+      </p>
+
+      <p class="ruled-top">
+        <strong>
+          How With-Action scenario was determines:
+        </strong>
+        <br />
+        {{@package.rwcdsForm.dcpHowdidyoudeterminethiswithactionscena}}
+      </p>
+    </section>
+
+    <section class="form-section">
+      <h2 class="section-header">
+        <span id="" class="section-anchor"></span>
+        Attachments
+      </h2>
+
+    </section>
+
+  </div>{{! end left/main column }}
+  <div class="cell large-4">
+
+    <StickySidebar
+      @navItems={{array
+        (hash anchorId="project-info"
+          label="Project Information")
+        (hash anchorId="project-description"
+          label="Description")
+        (hash anchorId="proposed-actions"
+          label="Proposed Actions")
+        (hash anchorId="analysis"
+          label="Analysis")
+        (hash anchorId="attachments"
+          label="Attachments")
+        }}
+    />
```

Comment:
I think we need to extract some of the less-form-related stuff so we can use the `saveableForm.Section` API

Reply:
Can we handle this is a separate PR? I just did this the same way it's done in the PAS form. But I did note that the StickySidebar needs some attention. We need to be able to use it more generally, not only for saveable forms. I want to refactor the StickySidebar component so that it's actually just a navigation component. The "sticky" part is just a class on its container, and we need to be able to render other things in that container.